manage_arkime/core/capacity_planning.py

```diff
@@ -3,13 +3,13 @@
 import logging
 import re
 import sys
-from typing import Dict, Type, TypeVar
+from typing import Dict, List, Type, TypeVar
 
 
 logger = logging.getLogger(__name__)
 
 MAX_TRAFFIC = 100 # Gbps, scaling limit of a single User Subnet VPC Endpoint
-MINIMUM_NODES = 1 # We'll always have at least one capture node
+MINIMUM_NODES = 2 # We'll always have at least two capture node for redundancy
```

Comment:
so for folks that want lab setting need to support 1

Reply:
> so for folks that want lab setting need to support 1

I'm not sure I agree on the cost/benefit analysis.  The cost difference in us-east-1 is like $30/mo which is unlikely to move the needle substantially.  But I think we have more important things to be focusing on right now and I don't have the bandwidth to make a more sophisticated and dedicated "lab" setting.  If you feel strongly about this, I'll just revert the change.